.github/workflows/node-pr.yml

```diff
@@ -7,6 +7,10 @@ on:
         description: "Node package manager to use"
         default: yarn
         type: string
+      is-yarn-berry:
+        description: "If Yarn Berry should be used"
+        default: false
```

Comment:
I think this should be flipped to `if-yarn-classic`. I think it makes sense to have modern yarn be the default

Reply:
Yep good callout :ok_hand: